disks-and-volumes.md

```diff
@@ -258,6 +258,43 @@ spec:
         claimName: mypvc
 ```
 
+### Ephemeral Volume
+
+Ephemeral volume is a volume that expects a persistent volume as it's backing
+store, but creates ephemeral image in front of it. All operations that would
+normally modify the volume contents are done against the ephemeral image,
+causing no permanent changes to the original volume. The ephemeral image is
+discarded when VM stops.
+
+Ephemeral volumes are great for testing environments, where VM images are
```

Comment:
I wouldn't call out 'testing' explicitly here. I think this feature has more potential than just testing. 

I'd say something along the lines that 'Ephemeral Volumes are useful in any scenario where disk persistence across VM restarts is not required"

Reply:
Actually had similar thoughts, the most important information would be giving several real-world use cases such as VM templates etc. But I think that's too far ahead given current state.

Reply:
it's fine not to explicitly call out real world scenarios.   I'd just avoid the key word "testing" because it could limit the feature's use.